doc/classes/Particles2D.xml

```diff
@@ -286,16 +286,16 @@
 	</methods>
 	<members>
 		<member name="amount" type="int" setter="set_amount" getter="get_amount">
-			Number of particles to emit.
+			Number of particles that can exist simultaneously and that are emitted in one emission cycle.
```

Comment:
This is redundant. You can just say "Number of particles emitted in one emission cycle."